src/tic_tac_toe/announcer.clj

```diff
@@ -6,10 +6,23 @@
   [player message]
   (conj message (str "\n"(:name player) "'s turn.\n")))
 
-  (defn move-request
-    [board]
-    (format human-move (count board)))
+(defn move-request
+  [board]
+  (format human-move (count board)))
+
+(defn player-win
+  [player]
+  (str (:name player) " wins!\n"))
+
+(defn winning-prompt
+  [status]
+  (-> "Game over. "
+      (cond->
+        (= :none (:winner status))
+          (str "It's a draw.\n")
+        (not= :none (:winner status))
+          (str (player-win (:winner status))))))
```

Comment:
`cond->` looks interesting. I am going to have to check that out!

I like to use indentation as a visual cue to the complexity of a function. Though I am not familiar with `cond->`, my inclination here would be to extract a function or two to reduce the indentation.

Reply:
[Updated this a bit.](https://github.com/saralein/tic-tac-toe_clojure/pull/6/files#diff-25c91f67a97e706c5a657b280e537c55R15)